draft-ietf-quic-recovery.md

```diff
@@ -228,6 +228,25 @@ QUIC endpoints measure the delay incurred between when a packet is received and
 when the corresponding acknowledgment is sent, allowing a peer to maintain a
 more accurate round-trip time estimate (see Section 13.2 of {{QUIC-TRANSPORT}}).
 
+### Probe Timeout Replaces RTO and TLP
+
+QUIC uses a probe timeout (see {{pto}}), with a timer based on TCP's RTO
+computation.  QUIC's PTO includes the peer's maximum expected acknowledgement
+delay instead of using a fixed minimum timeout. Unlike TCP, which collapses
+the congestion window upon expiry of an RTO, QUIC does not collapse the
+congestion window until persistent congestion ({{persistent-congestion}}) is
+declared and instead allows probe packets to temporarily exceed the congestion
+window whenever the timer expires.  In practice, this is similar to TCP with
+F-RTO, but it does allow more packets to be sent when the congestion window was
+not fully utilized prior to the probe timeout expiring. Though this is slightly
+more aggressive than TCP RTO, it's less aggressive than if the connection was
+not application limited.
```

Comment:
```suggestion
window whenever the timer expires. In doing this, QUIC avoids congestion window reductions that
might otherwise be caused by spurious retransmissions, obviating the need for correcting
mechanisms such as F-RTO {{cite}}.

Since QUIC does not collapse the congestion window on a PTO expiration, a QUIC sender is not
limited from sending more application data after a PTO expiration. A sender could be application-limited,
have a PTO timer expire, and then send as much application data as the congestion window allows after.
At first glance this might seem to be more aggressive than TCP's RTO mechanism, where a sender
is effectively limited to sending a single packet after an RTO expiration. However, TCP's RTO period is
more accurately modeled in QUIC by the persistent congestion event, after which QUIC also limits the
sender's congestion window. 
```

Reply:
I took most of this, but not all of it.